core/src/edu/cornell/gdiac/main/GameController.java

```diff
@@ -345,6 +362,26 @@ use setFrozen() (unimplemented) to freeze the player
       Make it so the player can tap a key to toggle the timer on and off
       Make the timer visually display whether it's enabled
       See my task in input controller */
+
+    if (!input.getMeterPaused()) {
+      metercounter += dt;
+
+      // If moving
+      if (input.getHorizontal() != 0 || input.getVertical() != 0) {
+        metercounter += stationaryconst;
+      }
```

Comment:
Currently holding any of WAD will continuously update the meter, which is not a big concern. In the future, we may choose to be more nuanced in what causes the meter to rise and by how much.

Reply:
yeah I agree, but for the gameplay prototype I think this is fine.